azure-iot-device/azure/iot/device/common/mqtt_transport.py

```diff
@@ -123,7 +125,7 @@ def _create_mqtt_client(self):
         def on_connect(client, userdata, flags, rc):
             logger.info("connected with result code: {}".format(rc))
 
-            if rc:
+            if rc:  # i.e. if there is an error
```

Comment:
>rc [](start = 15, length = 2)

I thought result code = 0 is a successful accepted connected

Reply:
That is correct.

`if rc:` is equivalent to `if rc != 0` (assuming `rc` is an integer). Thus the comment - it's already true, but I wanted to bring additional clarity.

---
In reply to: [326334174](https://github.com/Azure/azure-iot-sdk-python/pull/309#discussion_r326334174) [](ancestors = 326334174)